### PR TITLE
Fixed the issue #9 and updated the footer text alignment

### PR DESCRIPTION
**Pull request by Jaspreet099**
Hey @Iamdivyak I have fixed the changes 
- text alignment to left 
- Underline effect on hover

and also add hacktoberfest tag!

Screenshot 2023-10-18 at 11 38 39 AM

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -559,11 +559,14 @@ ul{
 }
 .quike-links ul li{
     margin-bottom: 25px;
+    /* padding-righ: 24px; */
+    margin-left: 120px;
```
Comment (Iamdivyak) [inline css/style.css]: this creates overlap in the mobile view
image

try another alternative.
thankyou

**Review by Iamdivyak — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (Jaspreet099) [reply css/style.css]: Okay, I didn't checked in Mobile view, Let me fix this

**Comment by Iamdivyak** [comment]
now it is not looking good on the desktop. 
image

Could you make it in the center as it was before? just change the hover effect. i will merge it

File: css/style.css (hunk 1 above)
Comment (Iamdivyak) [reply css/style.css]: delete the comments also

**Comment by Jaspreet099** [comment]
If i make it centre then on the mobile version it will overlap with icons, and change hover effect to?

File: css/style.css [hunk 2]
```diff
@@ -559,11 +559,14 @@ ul{
 }
 .quike-links ul li{
     margin-bottom: 25px;
+    /* padding-righ: 24px; */
+    /* margin-left: 120px; */
+    text-align: left;
```
Comment (Iamdivyak) [inline css/style.css:564]: delete this line

**Review by Iamdivyak — CHANGES_REQUESTED** [review]
delete line no: 562, 563 and 364

**Comment by Jaspreet099** [comment]
sure

**Pull request merged**